src/index.ts

```diff
@@ -1469,36 +1578,37 @@ class DevServer {
     options?: cp.SpawnOptions,
   ): Promise<cp.ChildProcess> {
     return new Promise<cp.ChildProcess>((resolve, reject) => {
-      const proc = this.spawn(command, args, cwd, { ...options, stdio: ['ignore', 'pipe', 'pipe'] });
-      const handleStream = (isStderr: boolean) => (data: Buffer) => {
-        let str = data.toString('utf-8');
-        str = str.replace(/\x1Bc/, ''); // filter the "clear screen" ANSI code (used by tsc)
-        if (str) {
-          str = str.trimEnd();
-          if (isStderr) {
-            console.error(str);
-          } else {
-            console.log(str);
+      this.spawn(command, args, cwd, { ...options, stdio: ['ignore', 'pipe', 'pipe'] }).then((proc) => {
```

Comment:
Why no await?

Reply:
because from what I know it is "bad practice" to make a "Promise function" an async method ... and I would need to do that in order to do await

Reply:
see also https://stackoverflow.com/questions/43036229/is-it-an-anti-pattern-to-use-async-await-inside-of-a-new-promise-constructor

Reply:
Ok, adjusted